tests/integration_tests/with_db/test_sequence_classification_single_text_script.py

```diff
@@ -12,7 +12,7 @@ def test_sequence_classification_single_text_script(
     for i in range(n_rows):
         input_data.append((
             bucketfs_conn_name,
-            model_params.sub_dir,
+            f"{model_params.sub_dir}",
```

Comment:
Using str() cast for converting to string, seems in this situation safer, because it is to add a other characters to the fstring